requirements_all.txt

```diff
@@ -177,3 +177,6 @@ pydispatcher==2.0.5
 
 # homeassistant.components.heatmiser
 heatmiserV3==0.9.1
+
+# homeassistant.components.seansor.dweet
```

Comment:
I can't use the script because of the issue with RPi.GPIO (as mentioned in #651).